doc/nrf/known_issues.rst

```diff
@@ -140,6 +140,11 @@ NCSDK-13895: Build failure for target Thingy:91 with secure_bootloader overlay
 Other issues
 ============
 
+.. rst-class:: v1-9-1 v1-9-0
+
+The time returned by date_time library is incorrect after one week of uptime
+  The time returned by date_time library is incorrect after one week has elapsed since the system booted because of an issue with clock_gettime() API.
```

Comment:
```suggestion
The time returned by :ref:`lib_date_time` library becomes incorrect after one week of uptime
  The time returned by :ref:`lib_date_time` library becomes incorrect after one week elapses. This is because the system boots due to an issue with clock_gettime() API.
```

Reply:
The system does not boot due to the issue. I tried to say that the issue becomes visible after device has been running continuously for one week. I think the first sentence is fine and the second sentence could be something like "This is because of an issue with clock_gettime() API."

Reply:
Updated as discussed offline.